src/stcal/tweakreg/astrometric_utils.py

```diff
@@ -170,6 +171,17 @@ def get_catalog(
     timeout : float, optional
         Timeout in seconds to wait for the catalog web service to respond. Default: 30.0 s
 
+    apply_pm : bool, optional
+        If True, apply proper motion when available. Otherwise,
+        do not apply proper motion at all. Default: True
```

Comment:
What would be the epoch?

Reply:
... if true?

Reply:
Not sure. As per ticket, True means whatever the pipeline is already doing. So what is jwst doing?

Reply:
What about selecting/using only those sources with proper motion information?

Reply:
> s-goldman pointed out alternative ways to filter at https://drizzlepac.readthedocs.io/en/latest/mast_data_products/pipeline-astrometric-calibration-description.html#fit-quality-and-selection-of-wcs-solution 

I don't think that link talks about selecting sources from a returned astrometric catalog.